### PR TITLE
Refactor TUI code and change interface selection

### LINKED ISSUE #18: stmgr-provision-hostconfig, UX change
Minor request for UX change. Selecting interfaces are clunky.

Today:
Tab down to "interfaces" select interface, select back

Wish:
Remove the interface button, and have a drop-down list similar to network-mode

### PR DESCRIPTION
**Pull request by MDr164**
Interfaces are now selectable on the main page of the UI via a dropdown menu. This allows only one interface to be selected at a time but simplifies the process of doing so. This resolves https://github.com/system-transparency/stmgr/issues/18

### CODE REVIEW

**Review by sylv-io — APPROVED** [review]

**Comment by MDr164** [comment]
Lets wait for a review from @bjoto to see if that change fits the desired UX.

**Comment by bjoto** [comment]
LGTM! I'll pull it in.

**Pull request merged**
